.github/workflows/main.yaml

```
@@ -0,0 +1,12 @@
+name: CI
+on:
+  pull_request:
+    branches: [main]
+jobs:
+  Awesome_Lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - run: npx awesome-lint
```

```suggestion
  Awesome_Lint:
    runs-on: ubuntu-latest
    steps:
      - uses: actions/checkout@v3
        with:
          fetch-depth: 0
      - run: npx awesome-lint

```

Using latest version of `checkout`

---

👍